miniapps/meshing/mesh-explorer.cpp

```diff
@@ -1250,6 +1252,25 @@ int main (int argc, char *argv[])
          cout << "New mesh file: " << omesh_file << endl;
       }
 
+      if (mk == 'D')
+      {
+         const char mesh_prefix[] = "mesh-explorer.mesh.";
+         MeshPartitioner partitioner(*mesh, np, partitioning);
+         MeshPart mesh_part;
+         int precision;
+         cout << "Enter desired precision: " << flush;
```

Comment:
"Precision" could mean different things, and the user may not be sure what it means here. Should we briefly clarify, e.g. "Enter desired precision (for decimal numbers in output)"?

Reply:
Updated to use the prompt "Enter floating point output precision (num. digits): ".